purchases/src/test/java/com/revenuecat/purchases/paywalls/components/ImageComponentTests.kt

```diff
@@ -12,29 +12,33 @@ import com.revenuecat.purchases.paywalls.components.properties.SizeConstraint
 import com.revenuecat.purchases.paywalls.components.properties.ThemeImageUrls
 import org.intellij.lang.annotations.Language
 import org.junit.Test
+import org.junit.experimental.runners.Enclosed
 import org.junit.runner.RunWith
 import org.junit.runners.Parameterized
 import java.net.URL
 
-@RunWith(Parameterized::class)
-internal class ImageComponentTests(@Suppress("UNUSED_PARAMETER") name: String, private val args: Args) {
```

Comment:
Reviewing this file using GitHub's "Hide whitespace" option makes it much clearer what has actually changed.